build.sh

```diff
@@ -12,6 +12,7 @@ usage()
   echo "Common settings:"
   echo "  --binaryLog                     Create MSBuild binary log (short: -bl)"
   echo "  --configuration <value>         Build configuration: 'Debug' or 'Release' (short: -c)"
+  echo "  --target-rid, -r <value>        Runtime identifier to be used by the build. e.g. alpine.3.18-arm64, fedora.37-x64, freebsd.13-arm64, ubuntu.19.10-x64"
```

Comment:
Suggestion to make more clear what this rid does, and what it does not do (https://github.com/dotnet/source-build/issues/4419):

```suggestion
  echo "  --rid, -r <value>               Overrides the rid that is produced by the build. e.g. alpine.3.18-arm64, fedora.37-x64, freebsd.13-arm64, ubuntu.19.10-x64"
```